app/(dashboard)/dashboard/statistics/page.tsx

```diff
@@ -10,6 +13,34 @@ export interface mockAttendant {
 }
 
 export default function Statistics() {
+  const [events, setEvents] = useState<Event[]>([]);
+  const [uid, setUid] = useState<string>('');
+  useEffect(() => {
+    const fetchEvents = async () => {
+      try {
+        const fetchedEvents = await queryForEvents();
+        if (fetchedEvents) {
+          // sort by descending order
+          console.log(fetchedEvents);
+          const sortedEvents = fetchedEvents.sort((a, b) => {
+            const dateA = parse(a.date, 'dd/MM/yyyy, HH:mm:ss', new Date());
+            const dateB = parse(b.date, 'dd/MM/yyyy, HH:mm:ss', new Date());
+            return compareDesc(dateA, dateB);
+          });
+          console.log(sortedEvents);
+          setEvents(sortedEvents);
+        }
+        const userId = (await getUserId()) ?? '';
+        if (userId !== '') {
+          setUid(userId);
+        }
+      } catch (error) {
+        console.error('Error querying for events:', error);
+      }
+    };
+
+    fetchEvents();
+  }, []);
   const attendants: mockAttendant[] = [
```

Comment:
do we still need this attendants variable :'0

Reply:
oh we don't, I'll remove it, good catch